components/Templates/includes/functions-view_template.php

```diff
@@ -329,13 +329,25 @@ function frontier_do_subtemplate( $atts, $content ) {
 		} elseif ( 'file' == $field['type'] && 'attachment' == $field['options']['file_uploader'] ) {
 			$template = frontier_decode_template( $content, $atts );
 			foreach ( $entries as $key => $entry ) {
```

Comment:
```suggestion
			$entry_pod = pods( 'media' );

			foreach ( $entries as $key => $entry ) {
```

Reply:
Doesn't this load all media items? I could imagine a big performance impact there.